### ISSUE TITLE
Restore a unique User-Agent

### ISSUE THREAD

**Issue by Aeyoun** [brave/browser-laptop#3693]
Brave should show some bravery of its own and own up to having its own User-Agent. I'm aware of and acknowledge [this issue](https://github.com/brave/browser-laptop/issues/242), but without identifying itself – brave will be counted in online statistics as just Chrome. Who knows? Maybe Brave will be more popular than Chrome one day, but website owners will not be prompted to sit up and take notice of Brave if they can't identify any traffic from the browser and its users.

**Comment by Sh1d0w** [comment]
In my opinion since Brave will block ads on the websites and most of website owners are not happy about it, by making unique user agent for Brave browser will just make easy for them to detect and block the website for users who use Brave. And so forcing them to switch to anogher browser if they want to browse the website.

**Comment by Aeyoun** [comment]
Yeah, but Brave shouldn’t default to be a bad actor who hides their identify and tries avoiding owning up to what it is. Brave can already be detected with scripting (and so can adblocking), so this doesn’t change things all that much.

**Comment by Sh1d0w** [comment]
I think if this gets implemented in Brave, it has to be made configurable, you know in Chrome and FF you can override the default user agent with one of your choice.

**Comment by Liunkae** [comment]
CC: @diracdeltas

**Comment by diracdeltas** [comment]
We can add the option to set your own UA easily but we don't have plans to restore a unique UA until we reach some nontrivial percentage of browser share.

**Comment by FineHub** [comment]
@BrendanEich I'm writing here because @bsclifton block commentating there.

If you have not read about it, I had pinpointed Firefox users too on Reddit, not only Brave. The discussion had over 14,000 views in a day, and hundreds of comments, which explains how big the problem is. We understand User-Agent is not a reliable indicator. Our intention is not to stop every user, but most of the ones that block ads. The useragent is accessible for a power user, but not for a regular user. The API based blocking will not even be accessible to a power user. We dont want that to happen, as we respect the privacy for people who genuinely have a concern i.e: if they are in a hostile country, we dont track them by default, or if they are a journalist, we dont track them. 

The thing is making these things accessible to people who **dont** need it but use it just for the sake of using it, hurts the open web. 5% traffic using privacy & adblocking addons is not an issue. But when everyone uses it, the model becomes unsustainable. 

What happens in the end because of this is that we start moving users to native apps, or block them from accessing. With native app, all trackers would work right? There is not an easy way to block there. 
Making this too accessible means that it is a loss for the open web in the long term when publishers cannot stay afloat. **Brave Ads is not the solution to the problem.** 

A proper path for Brave should be to include an ad filter like Chrome that blocks intrusive ads, and blocks networks that do not respect DNT. No 3rd party cookie blocking by default, just blocking domains which do not respect DNT, and networks with intrusive Ads is a better and sustainable way forward for the open web. 

Ads are better than ever, remember the 2000's? Popups everywhere. What did we do? We implemented popup blockers. The popup blockers blocked bad popups, but did not block the genuine ones. Same analogy applies here. Not blanket banning but forcing the industry to improve, by punishing specifically the websites and networks that do not respect DNT and have intrusive advertising. 

Concerned,
FineHub

**Comment by grahamperrin** [comment]
Respectfully, 👎 for sidestepping [the lock](https://github.com/brave/browser-laptop/issues/14041#event-1613836678). Please consider a cooling-off period and salvaging https://redd.it/8hhss1

**Comment by da2x** [comment]
As a publisher, I’d really like to see how many visitors use Brave. Any chance Brave could open an API for getting aggregate statistics on usage by domain instead of a User-Agent?
